build/release-images.sh

```diff
@@ -29,6 +29,9 @@ CMD_TARGETS="${KUBE_SERVER_IMAGE_TARGETS[*]}"
 if [[ "${KUBE_BUILD_HYPERKUBE}" =~ [yY] ]]; then
     CMD_TARGETS="${CMD_TARGETS} cmd/hyperkube"
 fi
+if [[ "${KUBE_BUILD_CONFORMANCE}" =~ [yY] ]]; then
+    CMD_TARGETS="${CMD_TARGETS} vendor/github.com/onsi/ginkgo/ginkgo test/e2e/e2e.test cmd/kubectl"
```

Comment:
would it make sense to list these targets in `hack/lib/golang.sh` rather than repeating them in multiple places?

Reply:
i was not sure `KUBE_SERVER_IMAGE_TARGETS` was appropriate to change (see line 28)

Reply:
I meant to add another variable like `KUBE_CONFORMANCE_TEST_TARGETS` or something.

Reply:
perfect. doing that now